cocos/bindings/manual/jsb_cocos_manual.cpp

```diff
@@ -642,7 +642,7 @@ static bool js_engine_FileUtils_listFilesRecursively(se::State &s) {
         for (uint i = 0; i < static_cast<uint>(arg1.size()); i++) {
             list->setArrayElement(i, se::Value(arg1[i]));
         }
-        list->setProperty("length", se::Value(arg1.size()));
+        list->setProperty("length", se::Value(static_cast<std::uint32_t>(arg1.size())));
```

Comment:
static_cast will be optimized by compiler if builds to 32 bit, so there is no overhead for 32 bit build.